target_chains/cosmwasm/tools/src/helper.ts

```diff
@@ -0,0 +1,113 @@
+import { readFileSync, existsSync, mkdirSync } from "fs";
+import { rimrafSync } from "rimraf";
+import AdmZip from "adm-zip";
+import path from "path";
+import { DownloaderHelper } from "node-downloader-helper";
+
+// We have released the compile contacts on github. If a chain needs some specific
+// feature in a contract, a version of the contract with that specific features is
+// also released. For example, "injective.zip" for injective.
+// Given a contract version (eg 1.2.0) and  zipFileNames (eg ["injective", "osmosis"])
+// this method will return an object with key = zipFileName and value = compiledCode
+export async function getContractBytesDict(
+  artifactZipFileNames: string[],
+  version: string
+) {
+  const githubArtifactsLink = `https://github.com/pyth-network/pyth-crosschain/releases/download/pyth-cosmwasm-contract-v${version}/`;
+  const tmpCodeStorageDir = "./tmp";
+  // clear tmp directory before downloading contracts
+  rimrafSync(tmpCodeStorageDir);
+
+  if (existsSync(tmpCodeStorageDir) === false) {
+    mkdirSync(tmpCodeStorageDir, { recursive: true });
+  }
+
+  const uniqueArtifactsZipName = Array.from(new Set(artifactZipFileNames));
+
+  // download zip files
```

Comment:
I might be a bit paranoid, but can we also verify the checksum? If one of our gh accounts get compromised they can swap the binary without us knowing it.

Reply:
yeah agree with this

Reply:
hardcoded checksum